README.md

```diff
@@ -26,20 +26,25 @@ GitHub Actions only have access to the repository they run for. So, in order to
   This key should start with `-----BEGIN ... PRIVATE KEY-----`, consist of many lines and ends with `-----END ... PRIVATE KEY-----`. 
 4. In your workflow definition file, add the following step. Preferably this would be rather on top, near the `actions/checkout@v2` line.
 
-```yaml
-# .github/workflows/my-workflow.yml
-jobs:
-    my_job:
-        ...
-        steps:
-            - actions/checkout@v2
-            # Make sure the @v0.5.3 matches the current version of the
-            # action 
-            - uses: webfactory/ssh-agent@v0.5.3
-              with:
-                  ssh-private-key: ${{ secrets.SSH_PRIVATE_KEY }}
-            - ... other steps
-```
+   ```yaml
+   # .github/workflows/my-workflow.yml
+   jobs:
+       my_job:
+           ...
+           steps:
+               # This action has to precede ssh-agent, since it undoes its effects
+               - uses: actions/checkout@v2
```

Comment:
Maybe that is fixed in https://github.com/actions/checkout/pull/833, unless submodules are used?